android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -486,23 +456,25 @@ public void testLogFileConfigurationConstructors() {
     }
 
     @Test
-    public void testEditReadOnlyLogFileConfiguration() {
-        final File path = new File(
-                context.getCacheDir().getAbsolutePath(),
-                "testEditReadOnlyLogFileConfiguration"
-        );
+    public void testEditReadOnlyLogFileConfiguration() throws IOException {
+        final File path = tempFolder.newFolder();
         final String logDirectory = emptyDirectory(path.getAbsolutePath());
         LogFileConfiguration config = new LogFileConfiguration(logDirectory);
-        Database.log.getFile().setConfig(config);
```

Comment:
this was setting the config with a dynamic directory path and when the test ends the directory is getting removed. And when it tries to look for the directory in upcoming tests, which was failing due to `no file or directory - LiteCore error`.